packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -18,24 +22,35 @@ export default function PostMetadata({
   className,
   children,
   description,
+  isVideoType,
+  insaneMode,
 }: PostMetadataProps): ReactElement {
   const date = useMemo(
     () => createdAt && postDateFormat(createdAt),
     [createdAt],
   );
 
+  const timeActionContent = isVideoType ? 'watch' : 'read';
+
   return (
     <div
       className={classNames(
         'flex items-center text-theme-label-tertiary typo-footnote',
         className,
       )}
     >
+      {isVideoType && insaneMode && (
+        <PlayIcon secondary size={IconSize.XXSmall} className="mr-1" />
+      )}
       {!!description && <span>{description}</span>}
       {!!createdAt && !!description && <Separator />}
       {!!createdAt && <time dateTime={createdAt}>{date}</time>}
```

Comment:
Order of created at and readTime is swapped in design, checking with them if this is intended.

Reply:
It was like that for collections too, so I think its design inconsistency.